packages/sessions-sdk-react/src/components/explorer-link.tsx

```diff
@@ -0,0 +1,50 @@
+import { Network as FogoNetwork } from "@fogo/sessions-sdk";
+import type { ComponentProps } from "react";
+import { useMemo } from "react";
+import type { Link as UnstyledLink } from "react-aria-components";
+
+import { Link } from "./link.js";
+
+type Props = ComponentProps<typeof UnstyledLink> & {
+  network: FogoNetwork | SolanaNetwork;
+  txHash: string;
+};
+
+export const ExplorerLink = ({
+  children,
+  network,
+  txHash,
+  ...props
+}: Props) => {
+  const link = useMemo(() => mkLink(network, txHash), [network]);
```

Comment:
nit: `mkLink()` looks to be a function with a simple switch statement, so we can likely drop the `useMemo()` to save on memory